modules/Ticket/Evaluation.js

```diff
@@ -7,13 +7,16 @@ import * as Icon from 'react-bootstrap-icons'
 import { AppContext } from '../context'
 import { fetch } from '../../lib/leancloud'
 import { useMutation, useQueryClient } from 'react-query'
+import { useToggle } from 'react-use'
 
 export function Evaluation({ ticket, isCustomerService }) {
   const { t } = useTranslation()
   const { addNotification } = useContext(AppContext)
   const storageKey = `ticket:${ticket.id}:evaluation`
   const [star, setStar] = useState(ticket.evaluation?.star ?? 1)
-  const [content, setContent] = useState(localStorage.getItem(storageKey) ?? '')
+  const [content, setContent] = useState(ticket.evaluation?.content ?? localStorage.getItem(storageKey) ?? '')
+  const [editing, toggleEditing] = useToggle(ticket.evaluation === undefined)
```

Comment:
```suggestion
  const [editing, toggleEditing] = useToggle(!ticket.evaluation)
```
没有评价时 API 是返回 null 的，这里用 falsy 判断吧。